docs/extending_syzkaller.md

```diff
@@ -0,0 +1,39 @@
+# Extending syzkaller
+
+Here are the common files and some general edits that need to be made in order to extend syzkaller for a new OS kernel. However, there are some specific changes that may be required for a given kernel (For example gathering coverage from a given kernel, or some errors that might pop up and give a hint about what to tweak).
+
+## First general file
```

Comment:
Executor is important, but I am not sure it's a good title for a section. If it describes executor changes, then call it "Executor".
Or, perhaps it's better to not invent names, but rather call each section according to the dir name (e.g., "syz-executor", "pkg/build", it makes it clear where to look for these files)